### PR TITLE
Env vars

### LINKED ISSUE #63: adding environment variables
similar to how codecov-bash allows you to do

```bash
bash <(curl -s https://codecov.io/bash) -e FOO,BAR,SPAM
```

Could either be:

```
env_variables: [FOO, BAR, SPAM]
```
or a string using github actions substitutions:

```
variables: ${{ format('FOO=xxx,BAR={0},SPAM={1}', matrix.bar, matrix.spam) }}
```

---

Until this is implemented, is there any work around this? I really need to add env variables to uploads.

### PR DESCRIPTION
**Pull request by samuelcolvin**
fix #63

This can be seen working [here](https://github.com/samuelcolvin/pydantic/pull/1326/checks?check_run_id=521778929) (see the windows and macos builds).

### CODE REVIEW

**Comment by ibrahim0814** [comment]
Great idea! This PR LGTM, will merge and release a new version of this Action next week.

**Comment by samuelcolvin** [comment]
I've just noticed a typo `take` -> `tag` in two places. I'll fix tomorrow.

**Comment by ibrahim0814** [comment]
Thanks!

**Comment by samuelcolvin** [comment]
done, vscode has also got bossy and striped trailing whitespace which shouldn't have been there anyway.

**Comment by samuelcolvin** [comment]
@ibrahim0814 any chance this could get merged and released?

**Comment by samuelcolvin** [comment]
@ibrahim0814 bump.

What's happening here?

Is codecov is still operating?

I'm making an open source contribution to a commercial company! Have the decency and good sense to reply in days not weeks.

**Comment by samuelcolvin** [comment]
@ibrahim0814 @stevepeak this is very frustrating, what's happening here?

**Comment by stevepeak** [comment]
@samuelcolvin I apologize for the lack of response here. @drazisil and @thomasrockhu can jump on this. I have since stepped down from Codecov to work on other projects. Thank you!

**Comment by thomasrockhu** [comment]
Hi @samuelcolvin I appreciate your patience and lack of response. I'll take a look at this today/tomorrow.

File: index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ try {
   const token = core.getInput("token");
   const flags = core.getInput("flags");
   const file = core.getInput("file");
+  const env_vars = core.getInput("env_vars");
   fail_ci = core.getInput("fail_ci_if_error").toLowerCase();
```
Comment (thomasrockhu) [inline index.js:13]:
```suggestion
  const env_vars = core.getInput("env_vars");
  
  fail_ci = core.getInput("fail_ci_if_error").toLowerCase();
```

**Comment by vivodi** [comment]
What is the purpose of `env_vars`? Can it be viewed through Codecov's UI? Why do I feel that setting it makes no difference?

**Pull request merged**